.meteor/packages

```diff
@@ -93,3 +93,4 @@ practicalmeteor:sinon
 
 
 # Custom Packages
+shell-server
```

Comment:
was added by the meteor update to that (because new packages are always appended).  moved up into the "meteor" section to be more clear...